docker_registry_client_async/dockerregistryclientasync.py

```diff
@@ -900,12 +900,13 @@ async def head_blob(
             "result": client_response.status == HTTPStatus.OK,
         }
 
-    async def _head_manifest(self, image_name: ImageName, **kwargs) -> ClientResponse:
+    async def _head_manifest(self, image_name: ImageName, *, accept: str = None, **kwargs) -> ClientResponse:
```

Comment:
accidentally split the **kwargs

Reply:
It just a git-diff trick ...  I added `*, accept: str = None, ` to force accept to be passed by name only, instead of by position; but diff doesn't detect a change for the first character ...